packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -307,6 +325,73 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		);
 	});
 
+	itExpects(
+		"full initialization of data object should not happen by default",
+		[
+			{
+				eventName: "fluid:telemetry:Container:Request_cancel",
+				error: "Non interactive/summarizer client's data object should not be initialized",
+			},
+		],
+		async () => {
+			const TestDataObjectType1 = "@fluid-example/test-dataStore1";
+			const dataStoreFactory1 = new DataObjectFactory(
+				TestDataObjectType1,
+				TestDataObject1,
+				[],
+				[],
+			);
+			const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
+				[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
+			]);
+			const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore(
+				dataStoreFactory1,
+				registryStoreEntries,
+				undefined,
+				[],
+			);
+
+			// Create a container for the first client.
+			const container1 = await provider.createContainer(runtimeFactory);
+			await assert.doesNotReject(
+				requestFluidObject<TestDataObject1>(container1, "/"),
+				"Initial creation of container and data store should succeed.",
+			);
+
+			// Create a summarizer for the container and do a summary shouldn't throw.
+			const createSummarizerResult = await createSummarizerFromFactory(
+				provider,
+				container1,
+				dataStoreFactory1,
+				undefined,
+				ContainerRuntimeFactoryWithDefaultDataStore,
+				registryStoreEntries,
+			);
+			const summarizer = createSummarizerResult.summarizer;
```

Comment:
nit: Same comment as above. This variable is not required since its only used in one place. You can directly use `createSummarizerResult.summarizer` in the summarizerNow call below.

Reply:
Another option is to change the call to `createSummarizerFromFactory` to something like this:
`const { summarizer, container: summarizerContaier } = await createSummarizerFromFactory(...);`

Then you can use `summarizer` and `summarizerContainer` without having to create another variable.